src/Bicep.Cli/Commands/RootCommand.cs

```diff
@@ -98,6 +98,28 @@ private void PrintHelp()
       bicep build file.bicep --outfile file.json
       bicep build file.bicep --no-restore
 
+    {exeName} format [options] <file>
+    Formats a .bicep file.
+
+    Arguments:
+      <file>        The input file
+
+    Options:
+      --outdir <dir>        Saves the output at the specified directory.
+      --outfile <file>      Saves the output as the specified file path.
+      --stdout              Prints the output to stdout.
+      --newline             Set newline char. Valid values are ( Auto | LF | CRLF | CR ).
+      --indentKind          Set indentation kind. Valid values are ( Space | Tab ).
+      --indentSize          Number of spaces to indent with.
```

Comment:
>spaces [](http://example.com/codeflow?start=38&length=6)

When indent kind is set to tab, is it the number of spaces or number of tabs?

Reply:
Using indentKind and indentSize at the same time will result in an error stating:
`The --indentSize cannot be used when --indentKind is "Tab"`

I chose to do it that way since the prettyprinter ignores IndentSize if IndendKindOption is not space.
Ref:
https://github.com/Azure/bicep/blob/f41634486281c290a7438a394e97ff755ef843e2/src/Bicep.Core/PrettyPrint/PrettyPrinter.cs#L16

Maybe I should add `(Only valid with --indentSize set to Space)` to the end of the description?

Reply:
Yeah, that makes sense.

Reply:
Cool, I'll add the clarification.